examples/tutorials/run-scalable-flow-with-pipeline/pipeline.ipynb

```diff
@@ -0,0 +1,613 @@
+    "\n",
+    "## 1.1 Import the required libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import required libraries\n",
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "from azure.ai.ml import MLClient, load_component, Input, Output\n",
+    "from azure.ai.ml.constants import AssetTypes\n",
+    "from azure.ai.ml.dsl import pipeline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.2 Configure credential\n",
+    "\n",
+    "We are using `DefaultAzureCredential` to get access to workspace. \n",
+    "`DefaultAzureCredential` should be capable of handling most Azure SDK authentication scenarios. \n",
+    "\n",
+    "Reference for more available credentials if it does not work for you: [configure credential example](../../configuration.ipynb), [azure-identity reference doc](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity?view=azure-python)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    # Check if given credential can get token successfully.\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential not work\n",
+    "    credential = InteractiveBrowserCredential()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.3 Get a handle to the workspace\n",
+    "\n",
+    "We use 'config file' to connect to your workspace. Check [this notebook](../../configuration.ipynb) to get your config file from Azure ML workspace portal and paste it into this folder. Then if you pass the next code block, you've all set for the environment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get a handle to workspace\n",
+    "ml_client = MLClient.from_config(credential=credential)\n",
+    "\n",
+    "# Retrieve an already attached Azure Machine Learning Compute.\n",
+    "cluster_name = \"cpu-cluster\"\n",
+    "print(ml_client.compute.get(cluster_name))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. Load flow as component\n",
+    "If you’ve already authored a flow using the Promptflow SDK or portal, you can locate the flow.dag.yaml file within the flow folder. This YAML specification is essential for loading your flow into an Azure ML component."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "flow_component = load_component(\"../../flows/standard/web-classification/flow.dag.yaml\")"
```

Comment:
would you like to mention how azureml environment is supported for Flow component? You can refer to [this doc](https://microsoft.github.io/promptflow/cloud/azureai/use-flow-in-azure-ml-pipeline.html?highlight=flow+pipeline#environment-of-the-component).

Reply:
Good point. Will add more content here.